bf/bf.c

```diff
@@ -29,48 +26,81 @@ int main(int argc, char const *argv[])
 	else
 	{
 		FILE *bf_file = fopen(argv[1], "r");
-		FILE *c_output = fopen(argv[2], "w");
+		int unmatched_braces = 0;
 		char c;
-
-		fputs("/* generated by bf.c */#include <stdio.h>\nint main(void) {char s[30000]={0};char*sp=s;", c_output);
+		size_t size = 0;
+		char *op_buffer = malloc(EIGHT_KB);
+		assert(op_buffer != NULL);
 
 		while ((c = fgetc(bf_file)) != EOF)
 		{
+			if (size+14 % EIGHT_KB == 0)
```

Comment:
This will still fail. I don't think you want a modulo at all here, but a check against some kind of maximum variable.